src/cpython/context.jl

```diff
@@ -30,6 +30,46 @@ function _atpyexit()
     return
 end
 
+# By default, ensure libstdc++ in the Conda environment is compatible with
+# the one linked in Julia. This is platform/version dependent, so needs to
+# occur at runtime.
+#
+# Allow the user to override the default.  This is useful when the version
+# of libstdcxx linked in Julia is customized in the local installation of
+# Julia.
+#
+# To figure out cxx_version for a given Julia version, run
+#   strings /path/to/julia/lib/julia/libstdc++.so.6 | grep GLIBCXX
+# then look at
+#   https://gcc.gnu.org/onlinedocs/gcc-12.1.0/libstdc++/manual/manual/abi.html
+# for the highest GCC version compatible with the highest GLIBCXX version.
+function get_libstdcxx_version_bound()
+    # This list comes from: https://gcc.gnu.org/onlinedocs/libstdc++/manual/abi.html
+    # Start with GCC 4.8, as it's extremely difficult to build Julia with anything older
+    vers_mapping = Dict(
+        18 => v"4.8.0",
+        19 => v"4.8.3",
+        21 => v"5.1.0",
+        22 => v"6.1.0",
+        23 => v"7.1.0",
+        24 => v"7.2.0",
+        25 => v"8.1.0",
+        26 => v"9.1.0",
+        27 => v"9.2.0",
+        28 => v"9.3.0",
+        29 => v"11.1.0",
+        30 => v"12.1.0",
+        31 => v"13.1.0",
```

Comment:
Looking at https://gcc.gnu.org/develop.html#timeline there are more versions of GCC than in that other linked page. If I understand right, I think it means that we could bump the mappings to `28 => 9.5.0`, `29 => 11.3.0`, `30 => 12.2.0`.

Reply:
I went ahead and applied you suggestion.  If I understand correctly, it seems that your suggestion is safe since libstdc++ should be backwards compatible.

Reply:
The ABI Policy page says "If a particular release is not listed, it has the same version labels as the preceding release." I take that to mean that all releases >=9.3.0 and <10.1.0 have the same version labels.